### PR TITLE
Fixing spelling mistake in people.yml

### PR DESCRIPTION
**Pull request by mwinitch**

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -2098,7 +2098,7 @@ mwinitch:
       begin_month: 6
       end_year: 2020
       end_month: 8
-      organization: columbiau
+      organization: Columbia University
```
Comment (sbillinge) [inline db/people.yml:2101]: this is not a spelling mistake.  columbiau is an id that gets dereferenced in institutions.yml

Comment (mwinitch) [reply db/people.yml:2101]: Do I have to change it back to the original in my branch?

**Comment by sbillinge** [comment]
@mwinitch I will just close this PR.  You can simply delete your branch locally

**Comment by sbillinge** [comment]
@mwinitch I will just close this PR.  You can simply delete your branch locally

**Pull request closed without merging**